wandb: Currently logged in as: nreints. Use `wandb login --relogin` to force relogin
wandb: wandb version 0.15.4 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.14.0
wandb: Run data is saved locally in /gpfs/home2/nreints/MScThesis/code/wandb/run-20230703_144431-kpf4jhww
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run misunderstood-mountain-440
wandb: ‚≠êÔ∏è View project at https://wandb.ai/nreints/ThesisFinal2
wandb: üöÄ View run at https://wandb.ai/nreints/ThesisFinal2/runs/kpf4jhww
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
wandb: üöÄ View run misunderstood-mountain-440 at: https://wandb.ai/nreints/ThesisFinal2/runs/kpf4jhww
wandb: Synced 7 W&B file(s), 0 media file(s), 0 artifact file(s) and 1 other file(s)
wandb: Find logs at: ./wandb/run-20230703_144431-kpf4jhww/logs
Training on dataset: data_t(5,20)_r(5,20)_combiR_pNone_gNone
Testing on 7 datasets: ['bigBlocks_data_t(5,20)_r(5,20)_tennis_pNone_gNone', 'bigBlocks_data_t(5,20)_r(5,20)_none_pNone_gNone', 'data_t(5,20)_r(5,20)_none_pNone_gNone', 'bigBlocks_data_t(5,20)_r(5,20)_semi_pNone_gNone', 'bigBlocks_data_t(5,20)_r(5,20)_combiR_pNone_gNone', 'bigBlocks_data_t(5,20)_r(5,20)_full_pNone_gNone', 'data_t(5,20)_r(5,20)_combiR_pNone_gNone']
Focussing on identity: False
Using extra input: False
Using fr-fr as reference point.
----- ITERATION 1/10 ------
Number of train simulations:  1920
Number of test simulations:  480
The dataloader for data/data_t(5,20)_r(5,20)_combiR_pNone_gNone took 52.19809126853943 seconds.
-- Finished Train Dataloader --
The dataloader for data/bigBlocks_data_t(5,20)_r(5,20)_tennis_pNone_gNone took 13.39882493019104 seconds.
Traceback (most recent call last):
  File "/gpfs/home2/nreints/MScThesis/code/gru.py", line 295, in <module>
    model = model_pipeline(
  File "/gpfs/home2/nreints/MScThesis/code/utils.py", line 325, in model_pipeline
    ) = make(
  File "/gpfs/home2/nreints/MScThesis/code/utils.py", line 412, in make
    data_set_test = dataset_class(
  File "/gpfs/home2/nreints/MScThesis/code/dataset.py", line 29, in __init__
    self.collect_data()
  File "/gpfs/home2/nreints/MScThesis/code/dataset.py", line 35, in collect_data
    with open(f"{self.dir}/sim_{i}.pickle", "rb") as f:
FileNotFoundError: [Errno 2] No such file or directory: 'data/bigBlocks_data_t(5,20)_r(5,20)_none_pNone_gNone/sim_0.pickle'
srun: error: gcn65: task 0: Exited with exit code 1
srun: launch/slurm: _step_signal: Terminating StepId=3007815.0

JOB STATISTICS
==============
Job ID: 3007815
Array Job ID: 3007807_6
Cluster: snellius
User/Group: nreints/nreints
State: FAILED (exit code 1)
Nodes: 1
Cores per node: 18
CPU Utilized: 00:01:13
CPU Efficiency: 4.46% of 00:27:18 core-walltime
Job Wall-clock time: 00:01:31
Memory Utilized: 2.47 GB
Memory Efficiency: 0.00% of 0.00 MB
